### ISSUE TITLE
Suspicious state from syntax checker python-pycompile

### ISSUE THREAD

**Issue by CannedYerins** [flycheck/flycheck#635]
Hi.  There seems to be some sort of issue with the formatting of errors coming from python-pycompile.

The code 
```python
for foo in bar:
    []
```
where the brackets signify the cursor position produces this flycheck output.

`Suspicious state from syntax checker python-pycompile: Checker python-pycompile returned non-zero exit code 1, but no errors from output: Sorry: IndentationError: expected an indented block (test.py, line 2)
Checker definition probably flawed.`

This definitely seems to be a valid error.  My grasp of elisp is minimal so I can't really follow the checker's function to see why this is considered suspicious.  

`M-x flycheck-version` produces `Flycheck version: 0.24snapshot (package: 20150412.242)`.

**Comment by lunaryorn** [comment]
@CannedYerins Apparently the error regexp is incomplete.

I'd recommend you to use Pylint or flake8 for Python, though.

**Comment by lunaryorn** [comment]
@CannedYerins Totally forgot about this:  Can you tell me your Python version?

**Comment by CannedYerins** [comment]
`$ python --version
Python 2.7.9`

**Comment by lunaryorn** [comment]
@CannedYerins Should be fixed now.  Please check with the next MELPA build.

**Comment by omps** [comment]
Hi Lunaryorn,

I am facing similar issue with the my emacs.

i am using
Flycheck version: 0.24snapshot (package: 20150612.235)

and python version 
Python 2.7.6

**Comment by lunaryorn** [comment]
@omps That's a different problem, so please open a separate issue.  But before doing so, please check that `flake8` is generally working on your system.
